src/components/DataView/DataViewActionsRow/DataViewActionsRow.styled.ts

```diff
@@ -5,3 +5,13 @@ export const DataViewActionsRowWrapper = styled.div`
 	display: flex;
 	justify-content: space-between;
 `;
+
+export const RightControlsContainer = styled.div`
+	.custom-checkbox {
+		margin-right: 6px;
```

Comment:
I think this might be 14px because it seems the space between the checkbox and the bulk button should be 24px and the checkbox already has 10

Reply:
Actually, the space should be 20px and this needs to change to 4px because as you said we have the following:

10px checkbox + 4px extra + 2px margin of the bulk button + 4px padding of the bulk button = 20px

![image](https://user-images.githubusercontent.com/87880265/205328294-32d47a62-2723-4b9e-a5da-3d9ff56e773b.png)